website/docs/r/service_dictionary_items_v1.html.markdown

```diff
@@ -0,0 +1,212 @@
+  name = "demofastly"
+
+  domain {
+    name = "demo.ocnotexample.com"
+    comment = "demo"
+  }
+
+  backend {
+    address = "demo.ocnotexample.com.s3-website-us-west-2.amazonaws.com"
+    name = "AWS S3 hosting"
+    port = 80
+  }
+
+  dictionary {
+    name = local.dictionary_name
+  }
+
+  force_destroy = true
+}
+
+// This resource is dynamically creating the items from the local variables through for expressions and functions.
+resource "fastly_service_dictionary_items_v1" "project" {
+  service_id = fastly_service_v1.myservice.id
+  dictionary_id = {for d in fastly_service_v1.myservice.dictionary : d.name => d.dictionary_id}[local.dictionary_name]
+  items = {
+    for division in local.host_divisions:
+      division => format("%s.%s", division, local.host_base)
+  }
+
+}
+```
+
+Lifecyle ignore_changes usage:
+
+The following example demonstrates how the lifecycle ignore_change field can be used to suppress updates against the 
+items in a dictionary.  If an external means, (for example via the Fastly API) is used to manage items in a dictionary, then this will stop terraform realigning the remote state.
+
+```hcl
+...
+
+resource "fastly_service_dictionary_items_v1" "items" {
+    service_id = "${fastly_service_v1.myservice.id}"
+    dictionary_id = "${{for s in fastly_service_v1.myservice.dictionary : s.name => s.dictionary_id}[var.mydict_name]}"
+    items = {
+        key1: "value1"
+        key2: "value2"
+    }
+    
+    lifecycle {
+      ignore_changes = [items,]
+    }
+}
+
+
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `service_id` - (Required) The ID of the Service that the dictionary belongs to
+* `dictionary_id` - (Required) The ID of the dictionary that the items belong to
+* `items` - (Optional) A Map representing an entry in the dictionary, (key/value)
+
+
+## Attributes Reference
+
+[fastly-dictionary]: https://docs.fastly.com/api/config#dictionary
+[fastly-dictionary_item]: https://docs.fastly.com/api/config#dictionary_item
+
+## Import
+
+Dictionary items can be populated through the Fastly API and/or the console.  These items can then be managed by terraform through the use of the import command.
+Resources are imported into the terraform state through the use of an ID, and the fastly_service_dictionary_items_v1 resouce has an ID that is made up of the following format.
+
+```
+<serviceID>/<dictionaryID>
+```
+
+This is an example of the import command being applied to the resource named `fastly_service_dictionary_items_v1.items`
+
+```
+$ terraform import fastly_service_dictionary_items_v1.items <serviceID>/<dictionaryID>
+```
+
+If terraform is already managing remote dictionary items against a resource being imported then the user will be asked to remove it from the existing terraform state.  
+The following is an example of the terraform state command to remove the resource named `fastly_service_dictionary_items_v1.items` from the terraform state file.
```

Comment:
Please change "terraform" to "Terraform"

Reply:
Files were searched and updated where appropriate.